core/workspace_svg.ts

```diff
@@ -1212,7 +1141,7 @@ export class WorkspaceSvg extends Workspace implements IASTNodeLocationSvg {
    */
   translate(x: number, y: number) {
     if (this.useWorkspaceDragSurface_ && this.isDragSurfaceActive_) {
-      this.workspaceDragSurface_.translateSurface(x, y);
+      this.workspaceDragSurface_?.translateSurface(x, y);
```

Comment:
Why `?.` here instead of `!.`?  Was the old code wrong in assuming that `.workspaceDragSurface_` can't be `undefined` here?